en/release-notes/upgrade-guide-v2.1.md

```diff
@@ -0,0 +1,164 @@
+
+In the previous version, v2.0, __Cocos Creator__ had used a __Cocos3D__ based 3D renderer to officially introduce 3D support. The addition of 3D features can greatly enrich the expressiveness of 2D games, reduce the resource overhead of 2D games, and bring more imagination and ideas to expand games.
+However, it should be emphasized that the target users of __Cocos Creator__ is still 2D game development, and the **2D game is still the core** service of __Cocos Creator 2.x__. We will continue to optimize the development experience and basic performance of 2D games, and gradually increase the 3D functions suitable for integration into 2D games. Therefore, __Cocos Creator__ does not allow a pure 3D game, which is the goal of __Cocos3D__. __Cocos Creator's__ development path is closer to 2.5D.
+
+This is not the first time __Cocos__ has upgraded the engine from 2D to 3D. A few years ago, __Cocos2d-x__ expanded to the 3D space and accumulated a lot of experience. It also produced a very good game concept like *Sword and Home*. However, at that time, 3D functions were piled up on a very heavy 2D renderer, and there was no editor support. It was not widely recognized by the market. This time, __Cocos Creator__ is based on __Cocos3D's__ 3D renderer, which has a lot of 3D stability and guarantees a seamless editor experience.
+
+## Follow-up Version Notice
+
+**v2.0 will continue to be maintained, native platform performance will be optimized as soon as possible**
+
+__v2.0__ and __v2.1__ will grow together over a period of time.__v2.0__ will continue to be maintained, adding more performance optimizations and bug fixes. We will soon be beta testing v2.0.7. In v2.0.7 the rendering performance of the engine has been optimized, especially **Spine** and **DragonBones**. Also, the performance has been greatly improved.
+
+**v2.1 iterations will be more aggressive**
+
+Compared with __v2.0__, the upgrade cycle of __v2.1__ will mainly focus on 3D features. We recommend upgrading new projects with 3D requirements to this version, and establishing contact with us on our forums. This allows us and our 3D partners to provide support.Since v2.1 is a very new version, we will speed up the iteration and try to meet the needs of developers. In particular, **currently v2.1.0 version still does not support 3D scene editing, only the 3D parameters can be set in the property inspector**. We will be releasing **v2.1.1** in the near future to support 3D scene editing, material systems and optimizing the 3D development process.
+
+# Complete list of changes (compared to v2.0.5)
+
+## What's New
+
+**3D Node**
+In __v2.1__, newly created `Node` objects are still 2D by default. When needed, you only need to click the 2.5D button at the top right of the `Node` to switch to **3D mode**. After switching, you can edit the **z axis**. See [3D Node](https://docs.cocos2d-x.org/creator/2.1/manual/en/3d/3d-node.html).
```

Comment:
1. "newly created `Node` objects are still 2D by default". I think we can get rid of the "objects"
2. Please do not use absolute paths

Reply:
3. I meant to correct that. I will correct every place in the document.

Reply:
1. done.

2. If I remove __objects__ then I have to make `Node` plural.